### PR TITLE
feat(button-toggle): align with 2018 material design spec

### PR DESCRIPTION
**Pull request by crisbeto**
Aligns the button toggle component with the latest Material design spec.

angular_material_-_google_chrome_2018-07-30_22-03-24
angular_material_-_google_chrome_2018-07-30_22-45-06

### CODE REVIEW

**Comment by josephperrott** [comment]
@crisbeto looks like there is a lint error on this one

**Comment by crisbeto** [comment]
I've sorted out the lint warning.

File: src/lib/button-toggle/button-toggle.ts [hunk 1]
```diff
@@ -377,6 +412,16 @@ export class MatButtonToggle extends _MatButtonToggleMixinBase implements OnInit
...
 
+  /** The appearance style of the button. */
+  @Input()
+  get appearance(): MatButtonToggleAppearance {
+    return this.buttonToggleGroup ? this.buttonToggleGroup.appearance : this._appearance;
+  }
+  set appearance(value: MatButtonToggleAppearance) {
+    this._appearance = value;
+  }
+  private _appearance: MatButtonToggleAppearance;
```
Comment (jelbourn) [inline src/lib/button-toggle/button-toggle.ts]: Could you add docs for this in a follow-up PR?

Comment (crisbeto) [reply src/lib/button-toggle/button-toggle.ts]: Sure, I've put it on my list.

**Pull request merged**